### ISSUE TITLE
Need to be able to configure whether to continue running a program after attaching gdb to it

### ISSUE THREAD

**Issue by huili80** [microsoft/vscode-cpptools#1582]
It seems vscode automatically continues the running program after attaching gdb to it, unlike in a standalone gdb session running program would be stopped after attached.

Maybe it's already possible to configure in launch.json to NOT continue the program after gdb is attached, in that case can anyone tell me what that configuration is? Otherwise such ability would be nice to have.

**Comment by pieandcakes** [comment]
@huili80 At this time, that isn't possible. Can you tell me why this is important and different than hitting `Break All` or `pause` after you attached? As attaching is non-deterministic, I don't understand the difference.

**Comment by huili80** [comment]
You're right about this being the same as hitting ```pause```. In fact, what i currently do is: hit ```pause```, run a few gdb commands prepended with ```-exec```, then hit ```continue```. The difference is, unlike in a standalone gdb session, where i can use a customized function to attach to a process, which runs the extra gdb commands before continues the program. With vscode, i can't automate this anymore. ```setupCommands``` doesn't help with this because the symbols aren't loaded yet when setupCommands are run.

**Comment by pieandcakes** [comment]
@huili80 Its something we can consider. I know we continue on attach now after we set up the debugger so a flag here would be possible.
